integration-tests/http/__tests__/product/store/product.spec.ts

```diff
@@ -1,24 +1,88 @@
+import { IStoreModuleService } from "@medusajs/types"
+import {
+  ApiKeyType,
+  ModuleRegistrationName,
+  ProductStatus,
+} from "@medusajs/utils"
 import { medusaIntegrationTestRunner } from "medusa-test-utils"
 import {
   adminHeaders,
   createAdminUser,
 } from "../../../../helpers/create-admin-user"
 import { getProductFixture } from "../../../../helpers/fixtures"
-import { ModuleRegistrationName } from "@medusajs/utils"
-import { IStoreModuleService } from "@medusajs/types"
+import { createDefaultRuleTypes } from "../../../../modules/helpers/create-default-rule-types"
```

Comment:
I think it's better for the http tests to not depend on modules helpers. We can either just do the fixture inline, do it as part of the `defaultsWorkflow` when booting medusa (if that's what we want), or move the helper higher up the stack.

Maybe creating these as part of the `defaultsWorkflow` makes sense, WDYT?

Reply:
I'll create a ticket to address this when i'm back from vacation 🤞🏻